client/settings/disable-upe-modal/index.js

```diff
@@ -0,0 +1,137 @@
+import React, { useContext, useEffect } from 'react';
+
+import { __ } from '@wordpress/i18n';
+import { dispatch } from '@wordpress/data';
+import { Button, Dashicon, ExternalLink } from '@wordpress/components';
+import interpolateComponents from 'interpolate-components';
+
+/**
+ * Internal dependencies
+ */
+import './style.scss';
+import ConfirmationModal from 'components/confirmation-modal';
+import PaymentMethod from 'components/payment-methods-list/payment-method';
+import useIsUpeEnabled from 'settings/wcpay-upe-toggle/hook';
+import WcPayUpeContext from 'settings/wcpay-upe-toggle/context';
+
+const NeedHelpBarSection = () => {
+	return (
+		<div className="disable-modal-help-notice">
+			<Dashicon className="disable-help-icon" icon="info-outline" />
+			<p>
+				{ interpolateComponents( {
+					mixedString: __(
+						'Need help? Visit {{ docsLink /}} or {{supportLink /}}.',
+						'woocommerce-payments'
+					),
+					components: {
+						docsLink: (
+							// eslint-disable-next-line max-len
+							<ExternalLink href="https://docs.woocommerce.com/document/payments/additional-payment-methods/#introduction">
+								{ __(
+									'WooCommerce Payments docs',
+									'woocommerce-payments'
+								) }
+							</ExternalLink>
+						),
+						supportLink: (
+							// eslint-disable-next-line max-len
+							<ExternalLink href="https://woocommerce.com/contact-us/">
+								{ __(
+									'contact support',
+									'woocommerce-payments'
+								) }
+							</ExternalLink>
+						),
+					},
+				} ) }
+			</p>
+		</div>
+	);
+};
+
+const DisableUPEModalBody = ( { enabledMethods } ) => {
+	return (
+		<>
+			<p>
+				{ __(
```

Comment:
Looks like this is inaccurate when only "card" is enabled.
![Screen Shot 2021-07-22 at 3 33 08 PM](https://user-images.githubusercontent.com/273592/126705870-786512fd-5a9c-4d4c-8b62-444d1a435176.png)


The "card" payment method will not be removed from checkout.

Also part of the discussion in here: p1626872628205000/1626808338.199400-slack-C01RTUVR81K

> Card shouldn’t be displayed in the modal as it’s only the new payment methods that require the UPE

Reply:
Should be good to go now.

With other methods enabled:

![Screen Shot 2021-07-23 at 2 35 49 PM](https://user-images.githubusercontent.com/6723003/126782403-9ffbc9f2-5b46-4a37-8d18-880cbe9da837.png)

With only CC/Debit enabled:

![Screen Shot 2021-07-23 at 2 36 16 PM](https://user-images.githubusercontent.com/6723003/126782444-4e4ad62c-0c89-44af-8c10-09d0e49d2cc7.png)

With only 1 alternative enabled(cc/debit disabled):

![Screen Shot 2021-07-23 at 2 37 04 PM](https://user-images.githubusercontent.com/6723003/126782515-1469c572-4b9f-4a74-b82e-0a288e89b311.png)